manifests/s/StefHeyenrath/GitHubReleaseNotes/1.0.8.0/StefHeyenrath.GitHubReleaseNotes.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: StefHeyenrath.GitHubReleaseNotes
+PackageVersion: 1.0.8.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/StefH/GitHubReleaseNotes/releases/download/1.0.8.0/Setup.msi
+  InstallerSha256: 72631494A411447DAD239384FB45F19E9D6A4269F54EB930045B10BBE8154850
+  ProductCode: '{1D840FA8-29CC-47FB-B192-A932DE13480F}'
```

Comment:
```suggestion
  ProductCode: '{32249977-C6C1-40ED-B6A8-7084E69A0E7A}'
```

Reply:
@ItzLevvie 
This should just be changed into a random Guid?

Reply:
@StefH, this isn't a random GUID, it's where the installer places its uninstall entries in the registry. You can find out the ProductCode after installation of the package by running this command in "Windows PowerShell" (not PowerShell Core/7)

```pwsh
get-wmiobject Win32_Product | Sort-Object Name | Format-Table IdentifyingNumber, Name -AutoSize
```